### PR TITLE
NEP: Fixes from NEP36 feedback

### PR DESCRIPTION
**Pull request by stefanv**
This is in response to comments by @mattip on the mailing list.

### CODE REVIEW

**Comment by mattip** [comment]
Thanks. I proposed this be merged in a week in http://numpy-discussion.10968.n7.nabble.com/Acceptance-of-NEP-36-fair-play-tt49356.html

File: doc/neps/nep-0036-fair-play.rst [hunk 1]
```diff
@@ -100,15 +103,16 @@ Fair play rules
      '1.17.2+mycompany.15`
 
    We understand that minor patches are often required to make a
-   library work under a certain distribution.  E.g., Debian may patch
+   library work inside of a distribution.  E.g., Debian may patch
    NumPy so that it searches for optimized BLAS libraries in the
-   correct locations.  But we ask that no substantive changes are
-   made.
+   correct locations.  This is in order, but we ask that no
```
Comment (rgommers) [inline doc/neps/nep-0036-fair-play.rst:108]: "This is in order" sounds odd to me. Maybe just "This is fine"

**Review by rgommers — APPROVED** [review]
LGTM, thanks @stefanv

File: doc/neps/nep-0036-fair-play.rst (hunk 1 above)
Comment (stefanv) [reply doc/neps/nep-0036-fair-play.rst]: I changed it to "acceptable".

File: doc/neps/nep-0036-fair-play.rst [hunk 2]
```diff
@@ -26,31 +26,34 @@ troublesome, bothersome, and unacceptable.
...
-  against NumPy, distributions put in their own fixes, most often by
-  changing a default keyword value. As a result the NumPy API was
-  inconsistent across distributions.
-
-When issues arise in cases like these, our developers waste time
-identifying the problematic release, locating alterations,
-and determining an appropriate course of action.
+In addition, packages on the Python Packaging Index are sometimes
+named such that users assume they are sanctioned or maintained by
+NumPy.  We wish to reduce the number of such incidences.
```
Comment (rossbar) [inline doc/neps/nep-0036-fair-play.rst:38]:
```suggestion
NumPy.  We wish to reduce the number of such incidents.
```
I think "incident" is the appropriate word when referring to discrete events, but NBD either way - feel free to ignore.

**Review by rossbar — APPROVED** [review]
LGTM, thanks for putting this together @stefanv !

**Review by charris — APPROVED** [review]

**Comment by charris** [comment]
Thanks Stephan.

**Pull request merged**